google-java-format.hcl

```diff
@@ -0,0 +1,28 @@
+description = "Reformats Java source code to comply with Google Java Style.."
+runtime-dependencies = ["openjre-17.0.4.1_1"]
+binaries = ["google-java-format"]
+source = "https://github.com/google/google-java-format/releases/download/v{version}/google-java-format-{version}-all-deps.jar"
```

Comment:
Variables need to be in the form `${version}` (note the `$`)

Reply:
Oops. Fixed!